src/core/server/saved_objects/service/lib/repository.ts

```diff
@@ -438,6 +487,41 @@ export class SavedObjectsRepository {
         versionProperties = getExpectedVersionProperties(version);
       }
 
+      savedObjectWorkspaces = options.workspaces;
+
+      if (expectedBulkGetResult.value.method !== 'create') {
```

Comment:
Can we add more comments here for this logic?

Reply:
Sure, good advice.